Plugin: 激战2！！！

**插件名称：**

激战2！！！

**插件功能：**

通过激战2星岬岛来返回信息

**PyPI 项目名：**

nonebot-plugin-gw2

**插件 import 包名：**

nonebot-plugin-gw2

**插件项目仓库/主页链接：**

https://github.com/Umamusume-Agnes-Digital/nonebot_plugin_gw2

**标签：**

[{"label":"gw2","color":"#52ea5a"}]

# 📃 商店发布检查结果

> Plugin: 激战2！！！

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-gw2 的信息如下：
    name         : nonebot-plugin-gw2     
     version      : 0.0.1                  
     description  : gw2 plugin for NoneBot 
    
    dependencies
     - asyncio &gt;=3.4.3
     - httpx &gt;=0.23.3
     - nonebot-adapter-onebot &gt;=2.1.5
     - nonebot2 &gt;=2.0.0rc3
插件 nonebot-plugin-gw2 依赖的插件如下：
    
插件 nonebot_plugin_gw2 加载出错：
    03-31 17:59:58 [SUCCESS] nonebot | NoneBot is initializing...
    03-31 17:59:58 [INFO] nonebot | Current Env: prod
    03-31 17:59:58 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_gw2&quot;
    Traceback (most recent call last):
      File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
        valid = load_plugin(&quot;nonebot_plugin_gw2&quot;)
      File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
        return manager.load_plugin(module_path)
    &gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
        module = importlib.import_module(name)
      File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1004, in _find_and_load_unlocked
    ModuleNotFoundError: No module named &#x27;nonebot_plugin_gw2&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: gw2-#52ea5a。</li><li>✅ 项目 <a href="https://github.com/Umamusume-Agnes-Digital/nonebot_plugin_gw2">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-gw2/">nonebot-plugin-gw2</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -5074,5 +5074,20 @@
     "homepage": "https://github.com/mas-alone/nonebot-plugin-sayoroll",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot-plugin-gw2",
+    "project_link": "nonebot-plugin-gw2",
+    "name": "激战2！！！",
+    "desc": "通过激战2星岬岛来返回信息",
+    "author": "Umamusume-Agnes-Digital",
+    "homepage": "https://github.com/Umamusume-Agnes-Digital/nonebot_plugin_gw2",
+    "tags": [
+      {
+        "label": "gw2",
+        "color": "#52ea5a"
+      }
+    ],
+    "is_official": false
   }
 ]

